### PR TITLE
Configure kuiper

### PR DESCRIPTION
**Pull request by mirceacaprioru**

### CODE REVIEW

File: stage2/01-sys-tweaks/01-run.sh [hunk 1]
```diff
@@ -11,6 +11,9 @@ install -m 644 files/console-setup   	"${ROOTFS_DIR}/etc/default/"
 
 install -m 755 files/rc.local		"${ROOTFS_DIR}/etc/"
 
+install -m 644 files/iiod.service	"${ROOTFS_DIR}/lib/systemd/system/"
+
+
```
Comment (commodo) [inline stage2/01-sys-tweaks/01-run.sh:19]: nitpick: extra line;

File: stage2/01-sys-tweaks/files/iiod.service [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# iiod - Systemd init script
+#
+# Copyright (C) 2016 Analog Devices Inc.
```
Comment (commodo) [inline stage2/01-sys-tweaks/files/iiod.service:3]: more recent year?
2020?

File: stage2/01-sys-tweaks/files/passwd [hunk 3]
```diff
@@ -0,0 +1 @@
+���e�;��
```
Comment (commodo) [inline stage2/01-sys-tweaks/files/passwd:1]: does this work on every build?
does this depend on the system unique/random salt that gets generated on the first boot?
tbh: i am bit a vague on this;

**Comment by commodo** [comment]
a few minor comments;
only one that may need closer attention on the default passwd

File: stage2/01-sys-tweaks/files/passwd (hunk 3 above)
Comment (mirceacaprioru) [reply stage2/01-sys-tweaks/files/passwd:1]: this works on every build. I compared two values from different builds an they are completely the same

File: stage2/01-sys-tweaks/files/iiod.service (hunk 2 above)
Comment (mirceacaprioru) [reply stage2/01-sys-tweaks/files/iiod.service:3]: I thinks i used a default file from somewhere and did not change anything to it including the copyright year so will keep this one for now.

Comment (commodo) [reply stage2/01-sys-tweaks/files/iiod.service:3]: ack

**Comment by mirceacaprioru** [comment]
changelog v1 -> v2
- remove extra line
- add cosmetic touches (launch icon and wallpaper)

File: stage7/01-adi-kernel-dts/00-run.sh [hunk 4]
```diff
@@ -34,6 +34,16 @@ build_linux $DEFCONFIG_PI3 $KERNEL_IMG_PI3
...
+cp -f $WORK_DIR/$LINUX_DIR/arch/$ARCH/boot/dts/bcm2708-rpi-b.dtb $STAGE_WORK_DIR/rootfs/boot
+cp -f $WORK_DIR/$LINUX_DIR/arch/$ARCH/boot/dts/bcm2708-rpi-b-plus.dtb $STAGE_WORK_DIR/rootfs/boot
+cp -f $WORK_DIR/$LINUX_DIR/arch/$ARCH/boot/dts/bcm2708-rpi-cm.dtb $STAGE_WORK_DIR/rootfs/boot
+cp -f $WORK_DIR/$LINUX_DIR/arch/$ARCH/boot/dts/bcm2708-rpi-zero.dtb $STAGE_WORK_DIR/rootfs/boot
+cp -f $WORK_DIR/$LINUX_DIR/arch/$ARCH/boot/dts/bcm2708-rpi-zero-w.dtb $STAGE_WORK_DIR/rootfs/boot
+cp -f $WORK_DIR/$LINUX_DIR/arch/$ARCH/boot/dts/bcm2709-rpi-2-b.dtb $STAGE_WORK_DIR/rootfs/boot
+cp -f $WORK_DIR/$LINUX_DIR/arch/$ARCH/boot/dts/bcm2710-rpi-3-b.dtb $STAGE_WORK_DIR/rootfs/boot
+cp -f $WORK_DIR/$LINUX_DIR/arch/$ARCH/boot/dts/bcm2710-rpi-3-b-plus.dtb $STAGE_WORK_DIR/rootfs/boot
+cp -f $WORK_DIR/$LINUX_DIR/arch/$ARCH/boot/dts/bcm2710-rpi-cm3.dtb $STAGE_WORK_DIR/rootfs/boot
+cp -f $WORK_DIR/$LINUX_DIR/arch/$ARCH/boot/dts/bcm2711-rpi-4-b.dtb $STAGE_WORK_DIR/rootfs/boot
```
Comment (commodo) [inline stage7/01-adi-kernel-dts/00-run.sh:46]: would it be worth to do a wild-card rule?
`cp -f $WORK_DIR/$LINUX_DIR/arch/$ARCH/boot/dts/bcm27*.dtb $STAGE_WORK_DIR/rootfs/boot`
?
for RPi5 or what comes next this needs constant maintenance

File: stage7/02-cosmetic/00-run.sh [hunk 5]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash -e
+
+cp -f files/wallpaper.png "${ROOTFS_DIR}/usr/share/rpd-wallpaper"
+cp -f files/launch.png "${ROOTFS_DIR}/usr/share/raspberrypi-artwork"
+
+on_chroot << EOF
+sed -i 's+/usr/share/rpd-wallpaper/temple.jpg+/usr/share/rpd-wallpaper/wallpaper.png+g' /etc/xdg/pcmanfm/LXDE-pi/desktop-items-0.conf
```
Comment (commodo) [inline stage7/02-cosmetic/00-run.sh:7]: what happens when this changes later? and it's not `temple.jpg`
is there some other way to configure this without needed to explicitly know the file?

**Comment by mirceacaprioru** [comment]
changelog v2->v3
- use wildcard for dtbs
- use sed to replace the value of wallpaper=

**Pull request merged**